### DISCUSSION TITLE
Support for PostgreSQL 13

### DISCUSSION THREAD

**Discussion by driek-vhees** [sqlalchemy/sqlalchemy#6637] [Usage Questions]
Hello everyone 👋 

I am not sure where to post this, so apologies if this is not the right place 🙇 
(I have spent 1h+ searching for the answer but alas I was not able to find anything, so I decided to ask for help here)

For our next project we are considering FastAPI with SQLAlchemy as ORM for our PostgreSQL DB.
It seems that version 13 however is not "Fully tested in CI" (as can be [seen here](https://docs.sqlalchemy.org/en/14/dialects/postgresql.html)), but what is the current roadmap in regard to support of this version?
We choose to go for version 13 as it seems to have better performance, but other than that there are no specific features that we plan to use at this moment.

Any help, insights, thoughts, known issues, anything that could help us make a better/informed decision on how to move forward is appreciated. 

Many thanks in advance

**Comment by zzzeek** [comment]
hey there -

I have good news and better news.  the good news is that PostgreSQL server versions are extremely backwards compatible so in general I would not have any concern about using the next PostgreSQL version vs. that which we have published.  SQLAlchemy's interaction with these databases is typically very conservative, and the bigger concern is usually SQLAlchemy's use of newer APIs or server constructs that didn't exist in older versions.


the better news is that we have PG 13 running on our CI since november of last year so that document appears to be out of date, will update it now.

**Reply by driek-vhees** [reply]
@zzzeek Thank you very much for your answer and that is great news indeed 🎉
In this case we will move forward using SQLAlchemy 🥳 
You made our day with this news/insight 🍻  (also thanks to everyone involved of course) 

Have a great day and thanks again for the swift answer 🙇
